packages/react-instantsearch-dom-maps/src/__tests__/Provider.js

```diff
@@ -388,11 +388,7 @@ describe('Provider', () => {
 
       const wrapper = shallow(<Provider {...props}>{x => x}</Provider>);
 
-      expect(wrapper.instance().getChildContext()).toEqual({
-        [STATE_CONTEXT]: expect.objectContaining({
-          isRefineOnMapMove: true,
-        }),
-      });
+      expect(wrapper.props().value.isRefineOnMapMove).toBe(true);
```

Comment:
We can use a `Consumer` to access the value rather than a direct access on the `props`.